test/e2e/defaulting.go

```diff
@@ -88,6 +88,10 @@ var _ = Describe("Default", func() {
 							Name:          string(features.PushReconciler),
 							Configuration: v1beta1.ConfigurationDisabled,
 						},
+						{
+							Name:          string(features.RawResourceStatusCollection),
+							Configuration: v1beta1.ConfigurationEnabled,
```

Comment:
so by default this feature is enabled, right? But still one has to enable it per type, is that correct?

Reply:
This is part of the tests, we want to enable it in the e2e tests for testing purposes.